src/codec/postcard_1_0.rs

```diff
@@ -1,18 +1,59 @@
-use postcard_1_0::{from_bytes, to_allocvec, Error};
-use serde::{Deserialize, Serialize};
+//! [postcard 1.0](https://crates.io/crates/postcard/1.0.8) ·
+//! Enable the `postcard_1_0` feature and annotate your type with
+//! `native_model::postcard_1_0::PostCard` to have `native_db` use this crate.
+
+/// Used to specify the [postcard 1.0](https://crates.io/crates/postcard/1.0.8)
+/// crate for serialization & deserialization.
+///
+/// # Warning
+///
+/// `postcard` does not implement all [serde](https://crates.io/crates/serde)
+/// features. Errors may be encountered when using this with some types.
+///
+/// If you are encountering errors when using this codec on your types, try
+/// using the `rmp_serde_1_3` codec instead.
+///
+/// # Basic usage
+///
+/// After enabling the `postcard_1_0` feature in your `Cargo.toml`, use the
+/// [`with`](crate::native_model) attribute on your type to instruct
+/// `native_model` to use `PostCard` for serialization & deserialization.
+///
+/// Example usage:
+///
+<<<<<<< HEAD
+/// ```rust
+/// #[derive(Clone, Default, serde::Deserialize, serde::Serialize)]
+=======
+/// ```no_run
+/// # fn main() {
+/// use serde::{Deserialize, Serialize};
+/// use native_model::native_model;
+///
+/// #[derive(Clone, Default, Deserialize, Serialize)]
+>>>>>>> bded1fc (Turn off doc tests)
```

Comment:
@lu-zero Can you remove these diff characters?

Reply:
Now hopefully should work